packages/@coorpacademy-components/src/atom/button-link-icon-only/index.js

```diff
@@ -78,7 +80,8 @@ ButtonLinkIconOnly.propTypes = {
     download: PropTypes.bool,
     target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top'])
   }),
-  disabled: PropTypes.bool
+  disabled: PropTypes.bool,
+  type: PropTypes.oneOf(['review', undefined])
```

Comment:
Étrange `undefined`

Reply:
si je laisse PropTypes.string et ne le passe pas de props j'ai des erreurs (Failed prop type: Invalid prop `knob.value` supplied to `ObjectType`), je n'ai pas trouvé comment marquer une prop avec oneOf mais qui soit optionnel

Reply:
comme vu dans le point avc @esa-coorp, utiliser plutôt un size: 'responsive'